cypress/e2e/donor-address.cy.js

```diff
@@ -0,0 +1,44 @@
+describe('Donor address', () => {
+    beforeEach(() => {
+        cy.visit('/donor-address');
+    });
+
+    it('address can be looked up', () => {
+        cy.injectAxe();
```

Comment:
This and the call in the next test could be moved to the `beforeEach` block.

Reply:
because the page changes on submit it needs to be injected between page loads, so it seemed better to have them all consistently like this vs. having one in a beforeEach and then manually injecting after page loads

Reply:
Ahhh of course, form submits. Makes sense 👍